### PR TITLE
Support sending event notifications to clients via Server Side Events/WebSockets

### PR DESCRIPTION
**Pull request by chirino**

### CODE REVIEW

**Comment by KurtStam** [comment]
Cool, what is this trying to solve exactly?

**Comment by chirino** [comment]
@KurtStam this works in conjunction with https://github.com/redhat-ipaas/ipaas-ui/pull/186 to implement data model syncing across different browser sessions.  This way you avoid users from needing to reload since their view automatically update when a different user changes the model.

**Comment by chirino** [comment]
squashing and rebasing off master.

**Comment by chirino** [comment]
Added a ITCase to valid the SSE functionality.

File: runtime/src/test/java/com/redhat/ipaas/runtime/EventsITCase.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import java.util.concurrent.TimeUnit;
+
+import static com.redhat.ipaas.runtime.Recordings.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+/**
+ * Used to test the SimpleEventBus
+ */
+public class EventsITCase extends BaseITCase {
```
Comment (jimmidyson) [inline runtime/src/test/java/com/redhat/ipaas/runtime/EventsITCase.java]: This is awesome @chirino!

**Comment by jimmidyson** [comment]
@chirino This is some awesome work! [merge]

**Pull request merged**